notebooks-d2l/finetune_cnn_jax.ipynb

```diff
@@ -0,0 +1,816 @@
+{
```

Comment:
<p>Training is slow - is there a way to speed it up? (Larger batch size?)</p><p>Is there a way to silence this annoying warning?</p><pre class="ql-syntax" spellcheck="false">Using cache found in /root/.cache/torch/hub/pytorch_vision_v0.10.0
<code></code>`
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/probml-notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
If I increase the batch size (upto 32) then after some iterations the notebook will go out of GPU memory and give out error, that's why I preferred to keep it low, but I will surely investigate on this.

This annoying warning is totally related to the external repository I used, I will change the model and see if I am still getting this warning or not.